geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala

```diff
@@ -0,0 +1,46 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.text
+
+import java.util.regex.Pattern
+
+import org.apache.commons.lang.StringEscapeUtils
+import org.parboiled.scala._
+
+/**
+  * Base class por parboiled parsers that provides methods for string and number matching
```

Comment:
Poor flavor wha?  

No, I was thinking that "Base class por parboiled parsers" might should be "Base class for parboiled parsers".

I'm fine if we internationalize the docs.:)  @mdzimmerman what's an ETA on Polish?